__tests__/api.test.js

```diff
@@ -87,5 +87,29 @@ describe("API tests", ()=>{
             })
         })
     })
-    describe("", () => {}) 
+    describe("GET /api/articles/:article_id/comments", () => {
+        test("200: Responds with an array of comments for the given article ID", () => {
+            return request(app)
+            .get("/api/articles/1/comments")
+            .expect(200)
+            .then((response) => {
+                response.body.forEach((comment) => {
+                    expect(comment).toHaveProperty('comment_id', expect.any(Number))
+                    expect(comment).toHaveProperty('votes', expect.any(Number))
+                    expect(comment).toHaveProperty('created_at', expect.any(String))
+                    expect(comment).toHaveProperty('author', expect.any(String))
+                    expect(comment).toHaveProperty('body', expect.any(String))
+                    expect(comment).toHaveProperty('article_id', expect.any(Number))
+                });
+            })
+        })
```

Comment:
This is a good start, but body should have an appropriate key which holds the array. I would recommend comments and the name of the key.

We should also be asserting the article_id here as we want to ensure every comment is actually linked to the article we are querying for. 

We should also need to have a length check of some kind before the forEach to prevent a false passing test if the array is empty.